frontend/src/app/ui/components/sequencer/sequencer.component.html

```
@@ -25,12 +25,12 @@
       <span class="large user-select-none"> bpm</span>
       @if (historyLength > 0) {
         <button class="button play-sample-button undo">
-          <img alt="undo" aria-label="Undo" [ngSrc]="'assets/images/icons/undo.svg' | iconDarkMode" height="28" width="28">
+          <img alt="undo" aria-label="Undo" [ngSrc]="'assets/images/icons/undo.svg' | iconDarkMode" height="28" width="28" (click)="sequencerService.dispatch({ type: 'UNDO'})">
         </button>
       }
       @if (futureLength > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)